src/apps/dashboard/routes/_redirects.ts

```diff
@@ -36,5 +36,6 @@ export const REDIRECTS: Redirect[] = [
     { from: 'usernew.html', to: '/dashboard/users/add' },
     { from: 'userparentalcontrol.html', to: '/dashboard/users/parentalcontrol' },
     { from: 'userpassword.html', to: '/dashboard/users/password' },
-    { from: 'userprofiles.html', to: '/dashboard/users' }
+    { from: 'userprofiles.html', to: '/dashboard/users' },
+    { from: 'trickplayconfiguration.html', to: '/dashboard/playback/trickplay' }
```

Comment:
This is only for existing paths that have changed so no need to add it for a new page.